src/status_im2/setup/events.cljs

```diff
@@ -10,7 +10,8 @@
             [status-im.multiaccounts.login.core :as multiaccounts.login]
             [status-im.native-module.core :as status]
             [status-im.utils.keychain.core :as keychain]
-            [status-im2.navigation.events :as navigation]))
+            [status-im2.navigation.events :as navigation]
+            status-im.utils.logging.core))
```

Comment:
its better to add to `status-im.events` because its not migrated yet

Reply:
ok, but why did you remove it from `status-im.events` in the first place?

Reply:
by mistake :)

Reply:
got it, thanks

Reply:
thank you